Updates to Backup & Recovery section (#892)
BitcoinDesign/Guide: guide/daily-spending-wallet/backup-and-recovery/manual-backup.md

-->

{% include picture.html
   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/backing-up-a-wallet-manual.png"

>    retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/backing-up-a-wallet-manual@2x.png"

Reviewer: Should link to this the [deleting the recovery phrase](https://deploy-preview-892--bitcoin-design-site.netlify.app/guide/daily-spending-wallet/security/#deleting-the-recovery-phrase) section in the security page somewhere on this page

Author: Where within the text does it make the most sense?

Reviewer: Probably at the very end as it's the last step you'd take (if you take it at all). Maybe even segment it into an Advanced section like me and Christoph have done with sending / requesting page

   mobile = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/backing-up-a-wallet-manual-mobile.png"
   mobileRetina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/backing-up-a-wallet-manual-mobile@2x.png"
   alt-text = "Backing up a recovery phrase chapter header image"
   width = 1600
   height = 600
   layout = "full-width"
%}

# Manual backup

> The daily spending wallet described in this reference design is designed to make backup as easy as possible for the user. As a result, the default flow is one where the user opts-in to a cloud backup.

Reviewer: ```suggestion
This wallet  is designed to make backup as easy as possible for the user. As a result, the default backup method is one the user opts-in to [automatic cloud backups]({{ '/guide/daily-spending-wallet/backup-and-recovery/cloud-backup/' | relative_url }}).
```

Author: ![Screen Shot 2022-08-18 at 4 33 57 PM](https://user-images.githubusercontent.com/2414177/185489524-164d2de0-2d30-4e24-8e7c-ed4ffef964cf.png)

> However, some users may not be comfortable with the idea of their private key on a cloud server. So in accordance with the design principles of [self-custody]({{ '/guide/getting-started/principles/#self-custody' | relative_url }}) and [transparency]({{ '/guide/getting-started/principles/#transparency' | relative_url }}), this wallet also offers a manual backup option.

Reviewer: ```suggestion
However, some users may not be comfortable with the idea of their recovery phrase on a cloud server. So in accordance with the design principles of [self-custody]({{ '/guide/getting-started/principles/#self-custody' | relative_url }}) and [transparency]({{ '/guide/getting-started/principles/#transparency' | relative_url }}), this wallet also offers a manual backup option.
```

## A note about channel state

> The lightning channel state is subject to change frequently, particularly if the user truly uses the daily spending wallet every day as the name would imply. As a result, we can’t think about the channel state as being a one-time backup. Channel state must be backed-up everytime the state is updated.

Reviewer: ```suggestion
The lightning channel state will change frequently as the user makes and receives payments with this wallet. As a result, we can’t think about the channel state as being a one-time backup. Channel state must be backed-up every time the state is updated.
```
Also sometimes you use lightning, sometimes Lightning - - lets just go with lowercase throughout? @GBKS

Author: Lowercase lightning maximalism

Author: Done

> As a result, this wallet requires channel state to be backed up automatically to a cloud provider. Allowing for it to be backed up manually would create a false sense of security.

Reviewer: ```suggestion
As a result, this wallet requires channel state to be backed up automatically to a cloud provider on each payment. Allowing for it to be backed up manually would create a false sense of security and will likely result in user error.
```

> This trade-off is designed to protect the user. In the event that a 3rd party manages to obtain and decrypt the channel state from the cloud storage, there is not a lot they can do; they have no way of stealing the funds without the keys.

Reviewer: ```suggestion
This trade-off is designed to protect the user. In the event that a 3rd party manages to obtain and decrypt the channel state from the cloud storage, there is not a lot they can do; they have no way of stealing the funds without the recovery phrase.
```

- Which trade-off exactly? Wasn't super clear reading this
- I thought it was optionally encrypted? If it is encrypted, what by? (not mentioned in cloud backup page).

Reviewer: When a manual backup is made the state backups should be encrypted by the recovery phrase. As you need the recovery phrase to recover your wallet this adds extra security without any additional friction.

Author: Changed it to "requirement", not sure why I worded it "trade-off".

Author: Yeah, that would make sense to encrypt with the recovery phrase, I think. There's not really much that can even be done with the channel state if the user lacks the recovery phrase, so it makes sense to do it this way.

Author: Changed it to talk about encrypting with the recovery phrase

Reviewer: Sweet, LGMT. Yeah its mostly a privacy benefit but it adds no additional friction for users so why not

So when we talk about “manual backup” in regard to this wallet, we are strictly talking about manual backup of the [recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }}).

## Backing up the recovery phrase

> This user flow usually requires users to manually back up their 12 to 24 word recovery phrase by writing it down on a piece of paper and storing it in a safe (but memorable location). In the case that a user’s device breaks or is stolen, the user can recover their funds and wallet by correctly entering their recovery phrase. The [private key management]({{ '/guide/private-key-management/manual-backup/' | relative_url }}) section dives further into the technical details of this scheme.

Reviewer 2: ```suggestion
This user flow usually requires users to manually back up their 12 to 24 word recovery phrase by writing it down on a piece of paper and storing it in a safe (but memorable) location. In the case that a user’s device breaks or is stolen, the user can recover their funds and wallet by correctly entering their recovery phrase. The [private key management]({{ '/guide/private-key-management/manual-backup/' | relative_url }}) section dives further into the technical details of this scheme.
```

   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-next-steps.png"
   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-next-steps@2x.png"
   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-next-steps@2x.png"
   layout = "shadow"
   caption = "Explain what is about to happen and what the user should do."
   alt-text = "Screen explaining that the recovery phrase will be shown next and how to write it down"
   width = 250
   height = 541
%}

{% include picture.html
   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-recovery-phrase.png"
   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-recovery-phrase@2x.png"
   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-recovery-phrase@2x.png"
   layout = "shadow"
   caption = "Displays the recovery phrase with numbering."
   alt-text = "Screen displaying a 12 word recovery phrase"
   width = 250
   height = 541
%}

</div>

> ### Print template

Reviewer 2: Should we link out to this [Figma community file](https://www.figma.com/community/file/1110806582648546839) that contains a template? We could combine both of our designs in there.

Author: There's no point combining them, I'll just use yours. It might be nice to have a descriptor in the community file but a) we can do that later and b) that might be something we ask for developer consultation on.

Author: Done

{% include /tip/open.html label="Tip: Be Clear about Numbering" icon="info" color="blue" %}

Note that this wallet explicitly instructs users to number each word (e.g. 1. sand 2. purple 3. flower). This is important because they will be asked to confirm their recovery phrase in a later step. It’s a bit of a pain for users to count which word corresponds to a particular number if they are not numbered initially.

{% include /tip/close.html %}

<div class="image-slide-gallery">

{% include picture.html
   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-choice.png"
   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-choice@2x.png"
   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-choice@2x.png"
   layout = "shadow"
   caption = "Wallet prompts the user to setup a backup, and the user selects the manual backup toggle."
   alt-text = "Screen prompting the user to back up after they have received a payment, with a manual backup toggle at the bottom"
   width = 250
   height = 541
%}

{% include picture.html

>    image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-notes.png"

Reviewer 2: The modal content is so big, maybe it should have it's own dedicated screen. Might need it anyways on smaller phones.

Author: Done

   layout = "shadow"
   caption = "Explain how to use the print template and give the option to print it or download it."
   alt-text = "Screen showing a picture of a print template with print and download buttons"
   width = 250
   height = 541
%}

{% include picture.html
   image = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-print-download.png"
   retina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-print-download@2x.png"
   modalImage="/assets/images/guide/daily-spending-wallet/backup-and-recovery/manual-backup/manual-backup-print-download@2x.png"
   layout = "shadow"
   caption = "User continues down OS specific flow for printing or downloading a file."
   alt-text = "Screen showing a placeholder for a mobile printing UI"
   width = 250
   height = 541
%}

</div>

### Confirming a backup

This step is a great way to ensure and test that the user in question actually stored their recovery phrase in a way that allows them to access and recount it. This typically entails prompting them to recall the words, which can be done in multiple ways that are laid out below.

> {% include /tip/open.html label="Tip: Confirm User Understanding" icon="info" color="blue" %}

Reviewer 2: ```suggestion
{% include /tip/open.html label="Tip: Confirm user understanding" icon="info" color="blue" %}
```
